lib/sandbox/execute.js

```diff
@@ -58,7 +58,8 @@ module.exports = function (bridge, glob) {
             execution = new Execution(id, event, context, options),
 
             waiting,
-            timers;
+            timers,
+            tests = {};
```

Comment:
Let's initialize this in the `legacy-interface`

Reply:
@kamalaknn If we initialize this in legacy-interface, then during `finish/teardown` we won't be having a reference to this object.
Again the only thing common to `setup` and `finish/teardown` would be `execution` object.
We would then have to add this `tests` object in `execution` during `setup` and delete it manually during `finish/tearndown`.
Does that sound ok?